src/utils/github_updater.rs

```diff
@@ -186,7 +186,7 @@ mod test {
         assert!(fields.is_ok());
 
         let fields = fields.unwrap();
-        assert!(fields.description != "".to_string());
+        assert!(fields.description != "");
```

Comment:
Ditto, please revert formatting changes. These are good changes, if you put them in a separate PR they wouldn't be blocked by writing tests for the `map`s.